src/main/java/org/apache/cassandra/distributed/shared/AbstractBuilder.java

```diff
@@ -135,6 +145,14 @@ public int getDatadirCount()
         return datadirCount;
     }
 
+    public int getTokenCount() {
+        return tokenCount;
+    }
+
+    public boolean isAllowVnodes() {
```

Comment:
this is used in Cassandra trunk patch, if this is set we skip the test if vnode is present (defined by `getTokenCount()`)